src/ecalc/libraries/libecalc/common/libecalc/core/consumers/consumer_system.py

```diff
@@ -151,6 +158,7 @@ def evaluate(
                     operator.mul, [consumer_result.component_result.is_valid for consumer_result in consumer_results]
                 )
 
+                energy_usage.unit = consumer_results[0].component_result.energy_usage.unit
```

Comment:
ah, yes, here you pick up the unit. can you move the energy_usage initialization down here so we do not need to do it in 2 steps?

Reply:
Exactly :)